app/models/assignment_repo/creator.rb

```diff
@@ -132,6 +133,26 @@ def delete_github_repository(github_repo_id)
       true
     end
 
+    # Public: Clone the GitHub template repository for the AssignmentRepo.
+    #
+    # Returns an Integer ID or raises a Result::Error
+    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+    def clone_github_template_repository!
```

Comment:
This is confusing since `clone` means to pull a repository down from a remote, which isn't happening here. What about `create_repo_from_template!`?